integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -320,7 +322,143 @@ case class AlterTableSplitPartitionCommand(splitPartitionModel: AlterTableSplitP
   }
 }
 
-case class CreateTable(cm: TableModel, createDSTable: Boolean = true) extends RunnableCommand
+case class AlterTableDropPartition(alterTableDropPartitionModel: AlterTableDropPartitionModel)
```

Comment:
please comment different points between different partition type in drop partition command.